WPFHospitalEditor/Building.xaml.cs

```diff
@@ -44,12 +44,19 @@ private void populateBuildingFloors(List<MapObject> allBuildingObjects)
             }
         }
               
-        private String findFloor(MapObject mapObjectIteration)
+        public static String findFloor(MapObject mapObjectIteration)
         {
             String[] firstSplit = mapObjectIteration.Description.Split("&");
```

Comment:
bad naming, try to find some kind of a meaningful variable name.
IMO even a one-liner would be better than a variable name like this